pom.xml

```diff
@@ -243,11 +254,11 @@ limitations under the License.
             </goals>
             <configuration>
               <rules>
-                <requireSameVersions>
+              <!--  <requireSameVersions>
```

Comment:
please undo this, we should be pulling in the same version from the grpc-bom

Reply:
my bad, wasn't intending to push this change. Updated!